<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-TU76DCC" ignored="0" name="ForgotPassword" tests="1" failures="1" timestamp="31 Jul 2018 09:07:19 GMT" time="44.004" errors="0">
  <testcase name="ForgotPasswordTest" time="43.096" classname="SystemMailTracking.testcases.SessionSuite.ForgotPasswordTestCase">
    <failure type="java.lang.reflect.InvocationTargetException">
      <![CDATA[java.lang.reflect.InvocationTargetException
at SystemMailTracking.driver.DriverScript.executeKeywords(DriverScript.java:49)
at SystemMailTracking.testcases.SessionSuite.ForgotPasswordTestCase.ForgotPasswordTest(ForgotPasswordTestCase.java:28)
Caused by: java.lang.AssertionError: The following asserts failed:
Object Not Found MailinatorForgotPasswordMail_xpath,
Failed to reset the password
at SystemMailTracking.keywords.GenericKeywords.reportFailure(GenericKeywords.java:362)
at SystemMailTracking.keywords.AppKeywords.verifyResetPassword(AppKeywords.java:147)
... 32 more
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- ForgotPasswordTest -->
</testsuite> <!-- ForgotPassword -->
